packages/terra-i18n/tests/nightwatch/i18n-spec.js

```diff
@@ -12,6 +12,8 @@ const defaultTranslations = enTransaltions;
 
 const screenshot = require('terra-toolkit').screenshot;
 
+const pauseInms = 1000;
```

Comment:
If we start using pause all over the place it is going to make our functional tests get slower and slower.  Is there a way to wait for a new element to be visible in this case?